jfrog-cli/artifactory/commands/buildinfo/publish.go

```diff
@@ -225,6 +235,22 @@ func createExcludeFilter(pattern string) filterFunc {
 	}
 }
 
+func setBuildInfoPropertiesForArtifacts(servicesManager *artifactory.ArtifactoryServicesManager, buildInfo *buildinfo.BuildInfo, partials buildinfo.Partials) error {
+	var resultItems []rtclientutils.ResultItem
+	for _, partial := range partials {
+		switch {
+		case partial.Artifacts != nil:
+			for _, artifact := range partial.Artifacts {
+				resultItems = append(resultItems, rtclientutils.ResultItem{Path: artifact.Path})
+			}
+		}
+	}
+
+	props := "build.name=" + buildInfo.Name + ";build.number=" + buildInfo.Number
```

Comment:
We should not build the build properties string by ourselves, but use the existing *CreateBuildProperties* API to do it for us. It also tales care of adding the *build.timestamp* property to the string.